custom_components/hacs/sensor.py

```diff
@@ -97,3 +114,9 @@ def device_state_attributes(self):
                 }
             )
         return {"repositories": repositories}
+
+    async def async_added_to_hass(self) -> None:
+        """Register for status events."""
+        self.async_on_remove(
+            self.hass.bus.async_listen("hacs/status", self._update_and_write_state)
```

Comment:
This is overkill? that even is used too much
Currently, I don't think there are any good events for this, but in general, I like the idea.

Reply:
Ideally there would be a specific dispatch/event when repository changes `pending_update` status, but this seemed better than polling every 15 seconds since this effectively disappears from the profile in most cases:

<img width="303" alt="Screen Shot 2020-11-14 at 9 19 54 AM" src="https://user-images.githubusercontent.com/663432/99155293-9c87f500-265a-11eb-9137-06149f1c463a.png">

Reply:
I want to add a task/job manager, that could create a separate event when done with the different tasks, but until then I guess `status` is fine :+1: